db/migrate/20221102182102_add_uniqueness_of_completed_resources.rb

```diff
@@ -0,0 +1,7 @@
+class AddUniquenessOfCompletedResources < ActiveRecord::Migration[7.0]
+  def change
+    remove_index :completed_resources, :user_id
```

Comment:
In the future, you could rollback migrations that where corrected within the PR. But this works fine.